### PR TITLE
CSS updates for site title heading on the homepage

### LINKED ISSUE #1433: Site Title looks weird in small screen devices.
## Description
Open the below link on a small device (I have it open in Galaxy Fold):
https://learn.wordpress.org/
For the looking good we can reduce the site title font size.
```
.site-header.home .site-title{
 font-size: 2.5rem;
}
```

## Step-by-step reproduction instructions

1. Open the URL https://learn.wordpress.org/
2. Show this url in Small device(Galaxy Fold)
3. Show the Site Title Size.

## Expected behaviour
Site title is small in small device.

## Actual behaviour
Site Title is to large in small device. 

## Screenshots 
https://tinyurl.com/2lzpotsh

## Environment information
- Chrome
- Samsung Galaxy Fold.

Also I have created the tickets in Meta Trac [#6871](https://meta.trac.wordpress.org/ticket/6871).

### PR DESCRIPTION
**Pull request by wpankit**
I have changed the css to update the Site title font sizes on smaller screen devices - Ex: Samsung fold device.

Trac ticket: https://meta.trac.wordpress.org/ticket/6871

Closes: #1433

### CODE REVIEW

**Comment by jonathanbossenger** [comment]
Hi @ankitpanchal75 thank you for this PR.

I notice that you've included your PhpStorm `.idea` folder, could I ask you to remove that and update the PR. Thanks

**Comment by wpankit** [comment]
> Hi @ankitpanchal75 thank you for this PR.
> 
> I notice that you've included your PhpStorm `.idea` folder, could I ask you to remove that and update the PR. Thanks

Sorry, I have removed the folder from the PR. Please check it.

**Comment by webtechpooja** [comment]
Thanks @ankitpanchal7 for the update

File: wp-content/themes/pub/wporg-learn-2020/css/vendor/_overrides.scss [hunk 1]
```diff
@@ -36,7 +36,6 @@ section {
 /* ___ Homepage ___ */
 
 .site-header.home .site-title {
-	font-size: 3.10rem;
```
Comment (adamwoodnz) [inline wp-content/themes/pub/wporg-learn-2020/css/vendor/_overrides.scss:39]: Removing this means that the font-size increases on larger screens due to the legacy styles not being overridden. Could you try instead scoping this rule to only screens larger than the mobile breakpoint?

```
.site-header.home .site-title {
	font-weight: 400;
	line-height: normal;
	
	@media only screen and (min-width: $breakpoint-small) {
		font-size: 3.10rem;
	}
}
```

**Review by adamwoodnz — CHANGES_REQUESTED** [review]
Thanks for this, I think we need to amend this slightly so that we don't cause a regression on larger screen sizes.

File: wp-content/themes/pub/wporg-learn-2020/css/vendor/_overrides.scss (hunk 1 above)
Comment (wpankit) [reply wp-content/themes/pub/wporg-learn-2020/css/vendor/_overrides.scss:39]: @adamwoodnz Sorry for the delay, I have updated the CSS now.

**Review by adamwoodnz — APPROVED** [review]

**Pull request merged**